docs/source/index.rst

```diff
@@ -34,6 +34,17 @@ The plugin portion of ``ci_watson`` contains:
   the author of the test to use this environment setting properly.
 * ``_jail`` fixture to enable a test to run in a pristine temporary working
   directory. This is particularly useful for pipeline tests.
+* Optional ``pytest.ini``/``setup.cfg [tool:pytest]`` configuration items
```

Comment:
AFAIK, we don't use `pytest.ini`, so mentioning it would just confuse people. Might want to give a `setup.cfg` snippet, as not everyone adding tests would be `pytest` experts. If there is not enough space in a bullet point, feel free to break this out into a subsection.

Reply:
Agreed about the 'too late' aspect. There's no way for pytest to know if a particular env var is going to be needed by the project's test code that executes later.

I anticipate `hstcal` will use `pytest.ini` since there's no reason for it to have a `setup.cfg` file.

Reply:
Oh, dear. I keep forgetting HSTCAL is not Python but using Python to run tests. No way to stick those things in its `conftest.py`?

Reply:
Perhaps, but I haven't investigated it since the `pytest.ini`/`setup.cfg [tool:pytest]` approach works. Based on the pytest docs, it seems placing configuration values in `conftest.py` might also be counter to the intended use of hosting per-directory hook implementations.

Reply:
OK, then please expand your doc to include 2 different snippets -- 1 for `pytest.ini` and another for `setup.cfg`; Might also want to state it clearly that choose one or the other but don't do both in the same repo.